transport/src/main/java/io/netty/channel/ChannelFuture.java

```diff
@@ -1,187 +0,0 @@
-/*
- * Copyright 2012 The Netty Project
- *
- * The Netty Project licenses this file to you under the Apache License,
- * version 2.0 (the "License"); you may not use this file except in compliance
- * with the License. You may obtain a copy of the License at:
- *
- *   https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations
- * under the License.
- */
-package io.netty.channel;
-
-import io.netty.bootstrap.Bootstrap;
-import io.netty.util.concurrent.BlockingOperationException;
-import io.netty.util.concurrent.Future;
-import io.netty.util.concurrent.GenericFutureListener;
-
-import java.util.concurrent.TimeUnit;
-
-
-/**
- * The result of an asynchronous {@link Channel} I/O operation.
```

Comment:
I wonder if we want to move some of this javadoc... This looks like some useful documentation we don't want to loose.

Reply:
Copied it over. Not sure if the class links work still.